docs/docs/external/getting-started/interop/data-sharing.md

```diff
@@ -0,0 +1,100 @@
+---
+id: data-sharing
+title: Testing Interoperation Scenarios for Data Sharing
```

Comment:
Can this have `sidebar_label: Data Sharing`?

Reply:
Fixed